eos-updater-prepare-volume/eos-updater-prepare-volume

```diff
@@ -27,10 +27,9 @@ import gi
 gi.require_version('Flatpak', '1.0')
 gi.require_version('GLib', '2.0')
 gi.require_version('OSTree', '1.0')
-gi.require_version('EosUpdaterFlatpakInstaller', '0')
 gi.require_version('EosUpdaterUtil', '0')
 from gi.repository import Flatpak, GLib, OSTree  # noqa
-from gi.repository import EosUpdaterFlatpakInstaller, EosUpdaterUtil
+from gi.repository import EosUpdaterUtil  # noqa
```

Comment:
What does `# noqa` mean here?

Reply:
It stops linting tools like `python3-flake8` emitting a warning on this line. I can’t remember which warning they emit — something about `EosUpdaterUtil` not being a known part of `gi.repository` iirc.

Reply:
(We run `flake8` as part of `check-local`.)